src/rules/flake8_bugbear/rules/getattr_with_constant.rs

```diff
@@ -21,6 +21,10 @@ fn attribute(value: &Expr, attr: &str) -> Expr {
     )
 }
 
+fn is_mangled_private(id: &str) -> bool {
```

Comment:
Do you mind just moving to `src/python/identifiers.rs`, and adding this docstring (I just write it but hadn't gotten much further :))

```rust
/// Returns `true` if a string is a private identifier, such that, when the
/// identifier is defined in a class definition, it will be mangled prior to
/// code generation.
///
/// See: <https://docs.python.org/3.5/reference/expressions.html?highlight=mangling#index-5>.
```